lib/querystring.js

```diff
@@ -212,12 +212,13 @@ QueryString.parse = QueryString.decode = function(qs, sep, eq, options) {
   sep = sep || '&';
   eq = eq || '=';
 
-  const obj = {};
-
   if (typeof qs !== 'string' || qs.length === 0) {
-    return obj;
+    return {};
   }
 
+  var obj = {};
```

Comment:
Sorry, had this open for a while and GH refreshed so I just now noticed this was already discussed.